eodag/resources/providers.yml

```
@@ -1254,7 +1258,7 @@
   url: https://mundiwebservices.com/
   search: !plugin
     type: QueryStringSearch
-    api_endpoint: 'https://{collection}.browse.catalog.mundiwebservices.com/opensearch'
+    api_endpoint: 'https://mundiwebservices.com/acdc/catalog/proxy/search/{collection}/opensearch'
```

```suggestion
    api_endpoint: 'https://{collection}.browse.catalog.mundiwebservices.com/opensearch'
```
restores https://github.com/CS-SI/eodag/pull/524